test/PowerShellEditorServices.Test/Language/SymbolsServiceTests.cs

```diff
@@ -256,10 +255,11 @@ public void FindsOccurrencesOnParameter()
         public async Task FindsReferencesOnCommandWithAlias()
         {
             List<SymbolReference> referencesResult = await GetReferences(FindsReferencesOnBuiltInCommandWithAliasData.SourceDetails).ConfigureAwait(true);
-            Assert.Equal(4, referencesResult.Count);
-            Assert.Equal("Get-ChildItem", referencesResult[1].SymbolName);
-            Assert.Equal("Get-ChildItem", referencesResult[2].SymbolName);
-            Assert.Equal("Get-ChildItem", referencesResult[referencesResult.Count - 1].SymbolName);
+            Assert.Collection(referencesResult,
+                (i) => Assert.Equal("Get-ChildItem", i.SymbolName),
+                (i) => Assert.Equal("gci", i.SymbolName),
```

Comment:
Why was this changed in https://github.com/PowerShell/PowerShellEditorServices/pull/1917 in the first place? Never made sense to me, but didn't investigate.